posts/Martin/2015-07-13-introduction-to-hibernate-search-genericjpa.adoc

```diff
@@ -0,0 +1,460 @@
+= Introduction to Hibernate Search with any JPA implementor (in this example: EclipseLink)
+Martin Braun
+:awestruct-tags: [ "Hibernate Search GenericJPA", "Hibernate Search", "JPA" ]
+:awestruct-layout: blog-post
+---
+Hibernate Search currently is only associated with Hibernate ORM/OGM.
+For many people this comes at no surprise as it already has “Hibernate” in its name.
+However during this year’s Google Summer of Code I am working on an integration of 
+Hibernate Search that aims to be able to be used with any JPA implementor 
+(so far Hibernate, EclipseLink and OpenJPA are supported). 
+
+In the following article I will show you how you can leverage the 
+power of Hibernate Search even though you chose not to use a 
+Hibernate based persistence provider in your application by using the new
+https://github.com/Hotware/Hibernate-Search-JPA[*Hibernate Search GenericJPA project*].
+
+== The example project
+
+Imagine a Book store (ok, a really simple one).
+Information about their books are stored in their MySQL database and accessed via EclipseLink.
+A sample persistence.xml and Maven pom.xml for that would look like this:
+
+[source,xml]
+----
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+	version="2.0">
+ 
+<persistence-unit name="EclipseLink_MySQL"
+	transaction-type="RESOURCE_LOCAL">
+	<provider>org.eclipse.persistence.jpa.PersistenceProvider</provider>
+	<class>org.hibernate.search.genericjpa.test.entities.Book</class>
+	<properties>
+		<property name="javax.persistence.jdbc.driver" 
+			value="com.mysql.jdbc.Driver" />
+		<property name="javax.persistence.jdbc.url" 
+			value="jdbc:mysql://localhost:3306/testingdb" />
+		<property name="javax.persistence.jdbc.user" 
+			value="hibernate_user" />
+		<property name="javax.persistence.jdbc.password"
+			value="hibernate_password" />
+		<property name="eclipselink.ddl-generation" 
+			value="drop-and-create-tables" />
+		<property name="eclipselink.logging.level" 
+			value="INFO" />
+		<property name="eclipselink.ddl-generation.output-mode"
+			value="both" />
+	</properties>
+</persistence-unit>
+ 
+</persistence>
+----
+
+[source,xml]
+----
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.hibernate</groupId>
+    <artifactId>hibernate-search-jpa-example</artifactId>
+    <version>0.2.0.0</version>
+    <packaging>jar</packaging>
+    <name>hibernate-search-jpa-example</name>
+ 
+    <build>
+        <plugins>
+            <plugin>
```

Comment:
I don't think that's needed for the example? I'd minimize it as much as posisible.